src/dropdown/dropdown.ts

```diff
@@ -15,10 +31,15 @@ import {NgbDropdownConfig} from './dropdown-config';
     '(document:click)': 'closeFromOutsideClick($event)'
   }
 })
-export class NgbDropdown {
+export class NgbDropdown implements AfterContentInit, OnDestroy {
   private _toggleElement: any;
 
   /**
+   * Placement of a dropdown. Accepts: "top", "bottom", "left", "right"
+   */
+  @Input() placement: 'top' | 'bottom' | 'left' | 'right' = 'bottom';
```

Comment:
This is in conflict with the existing `up` option. IMO we should not add the `placement` option (do we really need it on left or right???) but instead position as `top-left` / `bottom-left` based on the `up` flag being set or not.